src/data/projects.yml

```diff
@@ -81,7 +81,7 @@ categories:
     projects:
       - title: Gibbs Seed Company
         description: Gibbs Seed Company constantly monitors current agricultural conditions and appropriate seed hybrids for farmers to plant throughout their fields. Using Spark-Geo and PySAL they can analyze over 300 million planting options in under 10 minutes.
-        image: //developers.arcgis.com/assets/img/homepage/promo-python.png
+        image: /assets/img/projects/story-map-crowdsource-sm.png
```

Comment:
this image should actually be something from Spark-Geo or PySAL. (i forgot to update the placeholder `url`)

maybe we could reuse this?
https://github.com/mraad/spark-pip/blob/master/media/AOI.png